editoast/src/views/infra/mod.rs

```diff
@@ -473,13 +536,12 @@ async fn load(
     Ok(HttpResponse::NoContent().finish())
 }
 
-#[get("/cache_status")]
-async fn cache_status(
-    payload: Option<Json<StatePayload>>,
+/// Builds a Core infra load request, runs it
+async fn call_core_infra_state(
+    payload: Json<StatePayload>,
```

Comment:
```suggestion
    payload: StatePayload,
```

No need to leave it wrapped here, just unwrap it in the endpoints.